src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/paths/cookie.json

```diff
@@ -0,0 +1,48 @@
+{
+  "openapi": "3.1.0",
+  "info": [],
+  "paths": {
+    "/cookie-groups": {
+      "get": {
+        "tags": [
+          "Cookie Groups"
```

Comment:
Let's maybe just call it "Cookies".

Because then we can move all Cookies related Endpoints into that group.

For example next one would be something like this:
https://github.com/shopware/shopware/pull/11931